keyboards/lily58/keymaps/muuko/keymap.c

```diff
@@ -0,0 +1,188 @@
+/* Copyright 2020 @muuko
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+#include <string.h>
+
+#define IDLE_FRAMES 5
+#define IDLE_SPEED 20
+#define TAP_FRAMES 2
+#define TAP_SPEED 40
+#define ANIM_FRAME_DURATION 200
+#define ANIM_SIZE 512
+
+char wpm[16];
+uint32_t anim_timer = 0;
+uint32_t anim_sleep = 0;
+uint8_t current_idle_frame = 0;
+uint8_t current_tap_frame = 0;
+
+static long int oled_timeout = 300000;
+
+enum combos { ESCAPE_COMBO, DELETE_COMBO };
+const uint16_t PROGMEM escape_combo[] = { KC_GRV, KC_1, COMBO_END };
+const uint16_t PROGMEM delete_combo[] = { KC_DOT, KC_SLSH, COMBO_END };
+combo_t key_combos[COMBO_COUNT] = {
+    [ESCAPE_COMBO] = COMBO(escape_combo, KC_ESC),
+    [DELETE_COMBO] = COMBO(delete_combo, KC_DEL)
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_GRV,  KC_1,  KC_2,    KC_3,   KC_4,   KC_5,                   KC_6,   KC_7,    KC_8,    KC_9,   KC_0,    LCA_T(KC_MINS),
+        KC_TAB,  KC_Q,  KC_W,    KC_E,   KC_R,   KC_T,                   KC_Y,   KC_U,    KC_I,    KC_O,   KC_P,    RGUI_T(KC_EQL),
+        KC_LCTL, KC_A,  KC_S,    KC_D,   KC_F,   KC_G,                   KC_H,   KC_J,    KC_K,    KC_L,   KC_SCLN, RCTL_T(KC_QUOT),
+        KC_LSFT, KC_Z,  KC_X,    KC_C,   KC_V,   KC_B, KC_LBRC, KC_RBRC, KC_N,   KC_M,    KC_COMM, KC_DOT, KC_SLSH, RSFT_T(KC_BSLS),
+        KC_LGUI, MO(1), KC_RALT, KC_SPC,                                 KC_ENT, KC_BSPC, MO(1),   TG(1)
+    ),
+    [1] = LAYOUT(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,         KC_F11,
+        KC_TAB,  KC_MNXT, KC_HOME, KC_UP,   KC_PGUP, KC_VOLU,                   KC_PMNS, KC_P7,   KC_P8,   KC_P9,  KC_PSLS,        LCTL(KC_RGUI),
+        KC_LCTL, KC_MPLY, KC_LEFT, KC_DOWN, KC_RGHT, KC_VOLD,                   KC_PPLS, KC_P4,   KC_P5,   KC_P6,  KC_PAST,        RCTL_T(KC_PEQL),
+        KC_LSFT, KC_MPRV, KC_END,  KC_DEL,  KC_PGDN, KC_MUTE, KC_LPRN, KC_RPRN, KC_P0,   KC_P1,   KC_P2,   KC_P3,  C_S_T(KC_PDOT), KC_RSFT,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    )
+};
+
+void keyboard_post_init_user(void) {
+    wait_ms(100);
+}
```

Comment:
What's this for?

Reply:
thanks - turns out it wasn't necessary - removed